cli/dcoscli/data/help/node.txt

```diff
@@ -5,7 +5,8 @@ Usage:
     dcos node --help
     dcos node --info
     dcos node [--json]
-    dcos node log [--follow --lines=N --leader --master --mesos-id=<mesos-id> --slave=<slave-id>]
+    dcos node log [--follow --lines=N --leader --master --mesos-id=<mesos-id> --slave=<slave-id> --list-components]
```

Comment:
can we make list-components a new subcommand? (ie `dcos node list-components`) `dcos node log` is a little cluttered as is